libresapi/src/api/FileSharingHandler.h

```diff
@@ -48,10 +55,13 @@ class FileSharingHandler: public ResourceRouter
 
 	void handleDownload(Request& req, Response& resp);
 
-	StateToken mStateToken;
+	StateToken mLocalDirStateToken; // Token indicating change in local shared files
```

Comment:
Please use Doxigen style comments for headers documentation like https://github.com/RetroShare/RetroShare/blob/master/libretroshare/src/gxstrans/p3gxstrans.h#L210